src/activity_handlers.cpp

```diff
@@ -1028,26 +1028,26 @@ void activity_handlers::reload_finish( player_activity *act, player *p )
         return;
     }
 
+    std::string msg = _( "You reload the %s." );
+
     if( reloadable->is_gun() ) {
         p->recoil -= act->moves_total;
 
         if( reloadable->has_flag( "RELOAD_ONE" ) ) {
             for( int i = 0; i != qty; ++i ) {
```

Comment:
True. I suspect the compiler elides those and even if it doesn't it's unlikely to be a performance bottleneck. Code review is always welcome but as further changes to this function are likely in the imminent future we'd best tackle any such optimisations at the end.